test/RibbonThetaVaultWithSwap.ts

```diff
@@ -115,6 +115,33 @@ describe("RibbonThetaVaultWithSwap", () => {
     availableChains: [CHAINID.ETH_MAINNET],
     protocol: OPTION_PROTOCOL.GAMMA,
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon BNB Theta Vault (Call)`,
+    tokenName: "Ribbon BNB Theta Vault",
+    tokenSymbol: "rBNB-THETA",
+    asset: WETH_ADDRESS[chainId],
+    assetContractName: "IWETH",
+    strikeAsset: USDC_ADDRESS[chainId],
+    collateralAsset: WETH_ADDRESS[chainId],
+    chainlinkPricer: CHAINLINK_WETH_PRICER[chainId],
+    deltaFirstOption: BigNumber.from("1000"),
+    deltaSecondOption: BigNumber.from("1000"),
+    deltaStep: getDeltaStep("WETH"),
```

Comment:
we should add a `chainId === CHAINID.BINANCE` check in [getDeltaStep](https://github.com/ribbon-finance/ribbon-v2/blob/2b7c7e2c7958ccd6b3376e80cea1a77f821f0b1c/test/helpers/utils.ts#L699)`